buildSrc/src/main/kotlin/kotest-conventions.gradle.kts

```diff
@@ -0,0 +1,184 @@
+import org.gradle.api.tasks.testing.Test
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+import org.gradle.kotlin.dsl.all
+import org.gradle.kotlin.dsl.dependencies
+import org.gradle.kotlin.dsl.named
+
+plugins {
+  `java-library`
+  kotlin("multiplatform")
+  id("ru.vyarus.animalsniffer")
+}
+
+repositories {
+  mavenCentral()
+  mavenLocal()
+  gradlePluginPortal()
+}
+
+group = "io.kotest.extensions"
+version = Ci.version
+
+kotlin {
+  explicitApi()
+
+  targets {
+    metadata()
+
+    jvm {
+      compilations.all {
+        kotlinOptions.jvmTarget = "1.8"
+      }
+    }
+
+    js(IR) {
+      browser()
+      nodejs()
+    }
+
+    linuxX64()
+
+    mingwX64()
+
+    iosArm32()
+    iosArm64()
+    iosSimulatorArm64()
+    iosX64()
+    macosArm64()
+    macosX64()
+    tvosArm64()
+    tvosSimulatorArm64()
+    tvosX64()
+    watchosArm32()
+    watchosArm64()
+    watchosSimulatorArm64()
+    watchosX64()
+    watchosX86()
+  }
+
+  sourceSets {
+    val commonMain by getting
+    val commonTest by getting {
+      dependsOn(commonMain)
+    }
+
+    val jvmMain by getting {
+      dependsOn(commonMain)
+    }
+
+    val jvmTest by getting {
+      dependsOn(commonTest)
+      dependsOn(jvmMain)
```

Comment:
Nice. I thought it did so, but I didn't run into any problems when building with this configuration. Anyway, I'm all for removing redundant stuff :)

Reply:
Hmm, odd.. It started failing again with: 

```
 > Task :kotest-assertions-arrow-fx-coroutines:compileTestKotlinJs FAILED
e: /home/runner/work/kotest-extensions-arrow/kotest-extensions-arrow/kotest-assertions-arrow-fx-coroutines/src/commonMain/kotlin/io/kotest/assertions/arrow/fx/coroutines/runBlocking.kt: (6, 25): Expected function 'runBlocking' has no actual declaration in module <io.kotest.extensions:kotest-assertions-arrow-fx-coroutines_test> for JS
e: /home/runner/work/kotest-extensions-arrow/kotest-extensions-arrow/kotest-assertions-arrow-fx-coroutines/src/commonMain/kotlin/io/kotest/assertions/arrow/fx/coroutines/runBlocking.kt: (6, 25): Expected function 'runBlocking' has no actual declaration in module <kotest-assertions-arrow-fx-coroutines> for JVM
> Task :kotest-assertions-arrow-fx-coroutines:compileTestKotlinJvm FAILED
```

Sounds like the main source set is not available in the test compilation

Reply:
We're removing the runBlocking part in @nomisRev PR so maybe we can hold on until its merged and revisit, if there is no other option

Reply:
The PR might need a few days until it's merged. So we can go ahead with the original settings, if that gives us a green PR and we can merge this. 
Thanks a lot @Kantis :))

Reply:
Alright. I went back to the way things were.. I wouldn't have minded waiting, but I'm thinking it could've been an underlying problem that would manifest later on